app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,10 @@
+class PostsController < ApplicationController
+  def index
+    @user = User.find(params[:user_id])
+    @posts = Post.where(user_id: params[@user.id])
+  end
+
+  def display
+    @posts = Post.find(params[:id])
+  end
```

Comment:
- [ ] Based on the requirements we should have a `#show` action not `#display`. Kindly fix that. It is also a good idea to exhaust the inbuilt route actions provided by Rails before creating similar ones

Reply:
Thank you @TedLivist 
Let me fix that 👍🏾